### PR TITLE
fixing output of instance module

### PR DESCRIPTION
**Pull request by venturel**

### CODE REVIEW

**Comment by ringods** [comment]
Fixing by removing the output? Can you describe what problem this is fixing?

**Comment by venturel** [comment]
the parameter does not exist!

File: instance/outputs.tf [hunk 1]
```diff
@@ -34,10 +34,6 @@ output "instance_private_ip" {
   value = ["${concat(aws_instance.instance.*.private_ip, aws_instance.instance_no_ebs.*.private_ip)}"]
 }
 
-output "instance_vpc_security_group_ids" {
-  value = ["${concat(aws_instance.instance.*.instance_vpc_security_group_ids, aws_instance.instance_no_ebs.*.instance_vpc_security_group_ids)}"]
```
Comment (ringods) [inline instance/outputs.tf]: Don't remove this section, but update the parameter name from `instance_vpc_security_group_ids` to `vpc_security_group_ids`.

Comment (venturel) [reply instance/outputs.tf]: Why would you want that output?

Comment (venturel) [reply instance/outputs.tf]: Can you give me a use-case for that output?

**Pull request merged**